src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,55 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
int value is 0 by default